openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/Configurations.java

```diff
@@ -28,30 +28,49 @@
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public enum Configurations {
+public class Configurations {
 
-    V1_PNC_BUILDER_POD("v1_pnc-builder-pod.json"),
-    V1_PNC_BUILDER_SERVICE("v1_pnc-builder-service.json"),
-    V1_PNC_BUILDER_ROUTE("v1_pnc-builder-route.json"),
-    V1_PNC_BUILDER_SSH_SERVICE("v1_pnc-builder-ssh-service.json");
+    private static final String DEFAULT_V1_PNC_BUILDER_POD = "v1_pnc-builder-pod.json";
+    private static final String DEFAULT_V1_PNC_BUILDER_SERVICE_FILE = "v1_pnc-builder-service.json";
+    private static final String DEFAULT_V1_PNC_BUILDER_ROUTE_FILE = "v1_pnc-builder-route.json";
+    private static final String DEFAULT_V1_PNC_BUILDER_SSH_SERVICE_FILE = "v1_pnc-builder-ssh-service.json";
 
     private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass().getName());
 
     private static final String CONFIGURATIONS_FOLDER = "openshift.configurations/";
 
-    private final String filePath;
+    private static final String CUSTOM_V1_PNC_BUILDER_POD_PROPERTY = "v1_pnc-builder-pod-file";
```

Comment:
Drop "v1_" from the property names. The default files are v1, but user might specify v2 format (once available/supported).